src/nav2_msgs/msg/PathEndPoints.msg

```diff
@@ -1,3 +1,11 @@
 std_msgs/Header header
-geometry_msgs/Pose start
-geometry_msgs/PoseWithCovariance goal
+
+# The start pose for the plan
+geometry_msgs/PoseStamped start
```

Comment:
Why do we need a stamped version of the pose? We have the timestamp in the header already. Same for the goal.

Reply:
Currently not used. In future we might want to add time constraints to start and reach goal.

Reply:
Fixed, Mike is right, posestamp is not needed, and we don't want to overload the term. If we need to add time constraints we'll have to add a new field.